backend/src/main/java/fk/prof/backend/http/BackendHttpVerticle.java

```diff
@@ -78,6 +92,11 @@ private Router setupRouting() {
     router.put(ApiPathConstants.BACKEND_PUT_ASSOCIATION)
         .handler(this::handlePutAssociation);
 
+    router.post(ApiPathConstants.BACKEND_POST_POLL)
```

Comment:
Vertx routing APIs are really bad and I dislike the above usage as well. `router.(post/put/get/delete)` return a `Route` object which is in context of some uri path. Only a single handler can be registered on a route object or it overrides previous one and throws a warning at runtime. There are two ways to solve this:
1. Attach only a single handler. If you want to do multiple stuff, overload that handler with multiple responsibilities. In our case, it means re-implementing what bodyhandler gives us out of the box
2. Create multiple route objects for same uri path and attach handlers to each one of them. The handlers are called in the order they are attached